scripts/format.py

```diff
@@ -22,7 +22,8 @@
                  'release.h', 'genrand.cpp', 'address.cpp', 'visualizer_constants.hpp', 'icu-collate.cpp', 'icu-collate.hpp',
                  'yyjson.cpp', 'yyjson.hpp', 'duckdb_pdqsort.hpp', 'stubdata.cpp',
                  'nf_calendar.cpp', 'nf_calendar.h', 'nf_localedata.cpp', 'nf_localedata.h', 'nf_zformat.cpp',
-                 'nf_zformat.h', 'expr.cc', 'function_list.cpp']
+                 'nf_zformat.h', 'expr.cc', 'function_list.cpp', 'serialize_parsed_expression.cpp',
+                 'serialize_result_modifier.cpp', 'serialize_query_node.cpp']
```

Comment:
Could it make sense, in addition to the list, to have as part of the auto-generated header a line like: "PLEASE SKIP FROM FORMAT", and if that's found in any of the first 5 lines skip the file?

My guess most auto-generated files will over time end up breaking formatting, but can be moved further down the road.

Reply:
Good idea - should also be done for all auto-generated code.

Reply:
Have now added a skip for the formatting if the file contains `// This file is automatically generated by`